framework/src/org/checkerframework/framework/stub/StubParser.java

```diff
@@ -1389,8 +1390,6 @@ private AnnotationMirror getAnnotation(
 
     /*
      * Handles expressions in annotations.
-     * Supports String, int, and boolean literals, but not other literals
-     * as documented in the stub file limitation section of the manual.
```

Comment:
Can you add a link to that section in the manual?
Were class literals removed as restriction from that section? Should you add documentation about class literals?